package.json

```diff
@@ -344,7 +345,8 @@
     "os-name": "^2.0.1",
     "phantomjs": "1.9.8",
     "rimraf": "^2.5.2",
-    "should": "^9.0.2",
+    "should": "^11.1.1",
```

Comment:
we should not be upgrading modules not associated with this pr

Reply:
actually I recall we dropped `should` entirely!

Reply:
Must be a mistake, will fix.